RBAC/router.py

```
@@ -0,0 +1,348 @@
+    try:
+        result = await db.execute(
+            select(OrganizationUser.role).where(
+                OrganizationUser.user_id == current_user.id,
+                OrganizationUser.organization_id == organization_id
+            )
+        )
+        role = result.scalar()
+        if role is None:
+            raise HTTPException(status_code=404, detail="User role not found in the organization")
+        return {"role": role}
+    except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)